src/__tests__/Search/fetchRecentRevisions.test.tsx

```diff
@@ -39,52 +34,39 @@ describe('Search View/fetchRecentRevisions', () => {
       />,
     );
 
-    await screen.findAllByRole('button', { name: 'Base' });
-    await user.click(screen.getAllByRole('button', { name: 'Base' })[0]);
+    const baseRepoSelect = screen.getAllByRole('button', { name: 'Base' })[0];
+    expect(baseRepoSelect).toHaveTextContent('try');
+    await user.click(baseRepoSelect);
 
     // Menu items should be visible
     expect(
-      screen.getAllByRole('option', { name: 'autoland' })[0],
-    ).toBeInTheDocument();
-    expect(
-      screen.getAllByRole('option', { name: 'try' })[0],
+      screen.getByRole('option', { name: 'autoland' }),
     ).toBeInTheDocument();
+    expect(screen.getByRole('option', { name: 'try' })).toBeInTheDocument();
     expect(
-      screen.getAllByRole('option', { name: 'mozilla-central' })[0],
+      screen.getByRole('option', { name: 'mozilla-central' }),
     ).toBeInTheDocument();
 
-    await user.click(screen.getAllByRole('option', { name: 'autoland' })[0]);
-
-    act(() => void jest.runOnlyPendingTimers());
-    expect(screen.queryByText('mozilla-central')).not.toBeInTheDocument();
-
-    act(() => {
-      expect(store.getState().search[searchType].searchResults).toStrictEqual(
-        testData,
-      );
-    });
+    await user.click(screen.getByRole('option', { name: 'autoland' }));
+    expect(baseRepoSelect).toHaveTextContent('autoland');
 
     const searchInput = screen.getAllByRole('textbox')[0];
     await user.click(searchInput);
     await screen.findAllByText("you've got no arms left!");
-    expect(
-      screen.getAllByText("it's just a flesh wound")[0],
-    ).toBeInTheDocument();
```

Comment:
I removed this expectation because this is mostly a duplicate from the previous await.